src/main/java/org/assertj/assertions/generator/description/ClassDescription.java

```diff
@@ -102,6 +105,38 @@ public void addDeclaredFieldDescriptions(Set<FieldDescription> declaredFieldDesc
     this.declaredFieldsDescriptions.addAll(declaredFieldDescriptions);
   }
   
+  Stream<GetterDescription> getGetterStreamForField(FieldDescription base) {
```

Comment:
We should probably add the restriction to Maven. I forgot and IntelliJ let me do it, so I just went with it. :(